includes/Modules/AdSense.php

```diff
@@ -835,6 +849,32 @@ private function register_tag() {
 		}
 	}
 
+	/**
+	 * Checks if the module tag is found in the provided content.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string $content Content to search for the tags.
+	 * @return bool TRUE if tag is found, FALSE if not.
+	 */
+	public function has_placed_tag_in_content( $content ) {
+		$tag_matchers = ( new Tag_Matchers() )->regex_matchers();
+
+		$search_string = 'Google AdSense snippet added by Site Kit';
```

Comment:
It's important to note that this string is localized when printed on the front end, so if the site had a non-EN language, this would fail to match. It might be good to search for the untranslated and translated versions of the string to improve the chances of finding it.